src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/delete.js

```
@@ -0,0 +1,89 @@
+ *
+ * @author    Clement Gautier <clement.gautier@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+define(
+    [
+        'underscore',
+        'oro/translator',
+        'pim/form',
+        'text!pim/template/form/delete',
+        'oro/navigation',
+        'oro/loading-mask',
+        'routing',
+        'pim/dialog'
+    ],
+    function (
+        _,
+        __,
+        BaseForm,
+        template,
+        Navigation,
+        LoadingMask,
+        Routing,
+        Dialog
+    ) {
+        return BaseForm.extend({
+            className: 'btn-group',
+            template: _.template(template),
+            events: {
+                'click .delete': 'delete'
+            },
+            remover: {
+                remove: function(identifier) {
+                    throw 'remove function should be implemented in remover';
+                }
+            },
+            initialize: function (meta) {
+                this.config = _.extend({}, {
+                    trans : {
+                        title: 'pim_enrich.confirmation.delete_title',
+                        content: 'pim_enrich.confirmation.delete_content',
+                        success: 'pim_enrich.entity.info.deletion_successful',
+                        fail: 'pim_enrich.entity.info.deletion_failed'
+                    },
+                    redirect: 'oro_default'
+                }, meta.config);
+            },
+            render: function () {
+                this.$el.html(this.template());
+                this.delegateEvents();
+
+                return this;
+            },
+            delete: function () {
+                Dialog.confirm(
+                    __(this.config.trans.title),
+                    __(this.config.trans.content),
+                    this.doDelete.bind(this)
+                );
+            },
+            doDelete: function () {
```

agree with @grena but for now we can keep it like this. Ideally this method should throw an error